cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst

```diff
@@ -127,7 +127,7 @@ The previous command will compile ``.po`` files using
 After compiling the ``.po`` file(s), ``django-statici18n`` is used to create language specific catalogs. See
 ``django-statici18n`` `documentation <https://django-statici18n.readthedocs.io/en/latest/>`_ for more information.
 
-To upload translations to transiflex use the follow Make target::
+To upload translations to transifex use the follow Make target:
```

Comment:
Same here, `make push_translations` no longer exists as of OEP-58.

Reply:
Oh, so I'll remove all references to `push_translations` command.

Reply:
Please link to https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html instead.